WishList/WishList/Friends/SearchFriendsScreen.swift

```diff
@@ -12,7 +12,7 @@ struct SearchFriendsScreen: View {
     @State private var searchText : String = ""
     @State private var showInitialMessage = true
     
-    @State private var fullFakeFriends: [FakeFriend] = []
+    @State private var friends : [User] = []
```

Comment:
Change the name of the friends array for `people`, because they are might not be friends, so this name is not being accurate.